qiskit/quantum_info/operators/symplectic/stabilizer_table.py

```diff
@@ -990,7 +1201,40 @@ def _to_matrix(pauli, phase, sparse=False):
             csr_matrix: if sparse=True.
         """
         # pylint: disable=arguments-differ
-        mat = PauliTable._to_matrix(pauli, sparse=sparse, real_valued=True)
+        def count1(i):
```

Comment:
I recommend to count 1 by `bin(i).count('1')`. It's simpler and faster. https://github.com/Qiskit/qiskit-terra/pull/5261#discussion_r531359343

Reply:
Thank you. I use it.